help.lisp

```diff
@@ -54,11 +54,44 @@
                                   (:right (format nil "~a~a~a" (if (= c 0) "" padstr) len s)))))))
     (apply 'mapcar 'concat (or cols '(nil)))))
 
+(defun sort-flattened-bindings (keymaps sort-pred)
+  "Returns all bindings in KEYMAPS in a flattened list, globally sorted with SORT-PRED."
+  (let ((all-bindings (flatten (mapcar #'kmap-bindings keymaps))))
+    (sort all-bindings sort-pred)))
+
+(defun sort-by-alphabetical-command-p (b1 b2)
+  "Sort predicate that sorts bindings alphabetically by command name."
+  (not (null (string-lessp (binding-command b1) (binding-command b2)))))
```

Comment:
```suggestion
  (string-lessp (binding-command b1) (binding-command b2)))
```

`string-lessp` already returns a [generalized boolean](http://www.lispworks.com/documentation/HyperSpec/Body/26_glo_g.htm#generalized_boolean) so it is not necessary to wrap it with `(not (null ...))`